exporter/awsxrayexporter/internal/translator/cause_test.go

```diff
@@ -1014,3 +1014,105 @@ func TestParseExceptionPhpStacktraceMalformedLines(t *testing.T) {
 	assert.Equal(t, "test.php", *exceptions[0].Stack[1].Path)
 	assert.Equal(t, 89, *exceptions[0].Stack[1].Line)
 }
+
+func TestParseExceptionGoWithoutStacktrace(t *testing.T) {
+	exceptionType := "Exception"
+	message := "Thrown from grandparent"
+
+	stacktrace := ""
+
+	exceptions := parseException(exceptionType, message, stacktrace, "go")
+
+	assert.Len(t, exceptions, 1)
+	assert.NotEmpty(t, exceptions[0].ID)
+	assert.Equal(t, "Exception", *exceptions[0].Type)
+	assert.Equal(t, "Thrown from grandparent", *exceptions[0].Message)
+	assert.Nil(t, exceptions[0].Stack)
+}
+
+func TestParseExceptionGoWithStacktrace(t *testing.T) {
+	exceptionType := "Exception"
+	message := "error message"
+
+	stacktrace := `goroutine 19 [running]:
+go.opentelemetry.io/otel/sdk/trace.recordStackTrace(0x0, 0x0)
+	otel-go-core/opentelemetry-go/sdk/trace/span.go:323 +0x9b
+go.opentelemetry.io/otel/sdk/trace.(*span).RecordError(0xc0003a6000, 0x14a5f00, 0xc00038c000, 0xc000390140, 0x3, 0x4)
+	otel-go-core/opentelemetry-go/sdk/trace/span.go:302 +0x3fc
+go.opentelemetry.io/otel/sdk/trace.TestRecordErrorWithStackTrace(0xc000102900)
+	otel-go-core/opentelemetry-go/sdk/trace/trace_test.go:1167 +0x3ef
+testing.tRunner(0xc000102900, 0x1484410)
+	/usr/local/Cellar/go/1.16.3/libexec/src/testing/testing.go:1193 +0x1a3
+created by testing.(*T).Run
+	/usr/local/Cellar/go/1.16.3/libexec/src/testing/testing.go:1238 +0x63c`
+
+	exceptions := parseException(exceptionType, message, stacktrace, "go")
+	assert.Len(t, exceptions, 1)
+	assert.NotEmpty(t, exceptions[0].ID)
+	assert.Equal(t, "Exception", *exceptions[0].Type)
+	assert.Equal(t, "error message", *exceptions[0].Message)
+	assert.Len(t, exceptions[0].Stack, 5)
+
+	assert.Equal(t, "go.opentelemetry.io/otel/sdk/trace.recordStackTrace", strings.Split(*exceptions[0].Stack[0].Label, "(0x")[0])
```

Comment:
```suggestion
	assert.True(t, strings.HasPrefix(*exceptions[0].Stack[0].Label, "go.opentelemetry.io/otel/sdk/trace.recordStackTrace"))
```

Carry this through to the other instances.  It more accurately reflects what is being tested.